languages/english.py

```diff
@@ -386,8 +428,12 @@ def p_repeat_expr(p):
 def p_assign_expr(p):
     '''
     assign_expr : IDENTIFIER ASSIGN value_expr
+                | IDENTIFIER COINS
```

Comment:
Is this to handle n1=number of coins? Because where is the "="?
Also wouldn't the fix for this situation just be to have "number of coins" recognized as a token that returns a value?
Is that doable?

Reply:
= is part of the regular expression for COINS itself because '=' is not part of our language na